### PR TITLE
add ASP commit list

### PR DESCRIPTION
**Pull request by leecow**
add 2.1.1 changes mined from shiproom notes.

### CODE REVIEW

File: release-notes/2.1/2.1.1-commit.md [hunk 1]
```diff
@@ -35,4 +36,47 @@ This is a somewhat currated list of commits across the entire .NET Core 2.1.1 de
...
-* [`[68457365]`](https://github.com/dotnet/wcf/commit/68457365) Target svcutil.xmlserializer app at dotnetcore. (#2855)
\ No newline at end of file
+* [`[68457365]`](https://github.com/dotnet/wcf/commit/68457365) Target svcutil.xmlserializer app at dotnetcore. (#2855)
+
+## ASP
+
+### Entity Framework
+
+* [`[5bf9cc37]`](https://github.com/aspnet/EntityFrameworkCore/commit/5bf9cc370699f86e3609c183b3b23bfe2e3b1b23) Adds recursion guards to analyzer. [(#12054)](https://github.com/aspnet/EntityFrameworkCore/issues/12054)
+* [`[96049284]`](https://github.com/dotnet/corefx/pull/29846/commits/96049284dd577863c8b361fea06eadcfe5f1588b) Fix Sockets hang caused by concurrent Socket disposal [(#11993)](https://github.com/aspnet/EntityFrameworkCore/issues/11993)
```
Comment (ajcvickers) [inline release-notes/2.1/2.1.1-commit.md]: @leecow Two of these are commits from corefx--commits for EF issues are in the EF repo.

The commit for 11993 is: https://github.com/aspnet/EntityFrameworkCore/pull/12000/commits/a4a414432b20ae68ce53e4b981ecf9e718f98731

The commit for 11977 is: https://github.com/aspnet/EntityFrameworkCore/pull/12123/commits/9409a1f25b7603bb8dce747aff2a7df5744f5efe

File: release-notes/2.1/2.1.1-commit.md [hunk 2]
```diff
@@ -35,4 +36,45 @@ This is a somewhat currated list of commits across the entire .NET Core 2.1.1 de
...
+* [`[68457365]`](https://github.com/dotnet/wcf/commit/68457365) Target svcutil.xmlserializer app at dotnetcore. (#2855)
+
+## ASP
+
+### Entity Framework
+
+* [`[5bf9cc37]`](https://github.com/aspnet/EntityFrameworkCore/commit/5bf9cc370699f86e3609c183b3b23bfe2e3b1b23) Adds recursion guards to analyzer. [(#12054)](https://github.com/aspnet/EntityFrameworkCore/issues/12054)
+* [`[ef61273e]`](https://github.com/aspnet/EntityFrameworkCore/pull/12164/commits/ef61273ed948a2be8f0a081dc57b16ad3bc2de03) [(#12119)](https://github.com/aspnet/EntityFrameworkCore/issues/12119)
+* [`[1d447bdb]`](https://github.com/aspnet/EntityFrameworkCore/commit/1d447bdb0a550c52a04dcbd2cbc1200faa837686) SqlServer RevEng: Use concat operators instead of function[(#11932)](https://github.com/aspnet/EntityFrameworkCore/issues/11932)
+
```
Comment (ajcvickers) [inline release-notes/2.1/2.1.1-commit.md]: @leecow Looks like the two EF issues I commented on have now been removed. If this is all the commits in 2.1.1, then these two issues should still be here; they just had the wrong commits associated before.

Comment (leecow) [reply release-notes/2.1/2.1.1-commit.md]: I think the changes are largely captured by [7ce9270ac7] Fix Sockets hang caused by concurrent Socket disposal (#29786) (#29846)

**Comment by ajcvickers** [comment]
@leecow The sockets issue has nothing to do with the EF bugs.

**Comment by leecow** [comment]
Thanks Arthur, removed from the EF section.

**Pull request merged**